models/baseModels/SalesInvoice/SalesInvoice.ts

```diff
@@ -46,11 +73,55 @@ export class SalesInvoice extends Invoice {
         await posting.debit(discountAccount, discountAmount.mul(exchangeRate));
       }
     }
-
     await posting.makeRoundOffEntry();
     return posting;
   }
 
+  validations: ValidationMap = {
+    loyaltyPoints: async (value: DocValue) => {
+      if (!this.redeemLoyaltyPoints) {
+        return;
+      }
+      const partyDoc = (await this.fyo.doc.getDoc(
+        ModelNameEnum.Party,
+        this.party
+      )) as Party;
+
+      if ((value as number) <= 0) {
+        throw new ValidationError(t`Points must be greather than 0`, false);
+      }
```

Comment:
the second parameter (`false`) to `ValidationError` can be ignored. 

The error message should have more clarity like 
> Redemption Loyalty Points must be greater than 0